k8s/upgrade/src/plugin/objects.rs

```diff
@@ -338,6 +348,12 @@ pub(crate) fn upgrade_job(
                                 }),
                                 ..Default::default()
                             },
+                            EnvVar {
+                                // Ref: https://github.com/helm/helm/blob/main/cmd/helm/helm.go#L76
+                                name: "HELM_DRIVER".to_string(),
+                                value: Some(env::var("HELM_DRIVER").unwrap_or_default()),
```

Comment:
Now I'm confused, you made point about the cli having default value for the helm storage driver, but here you're ignoring the cli argument entirely and just using the env variable

Reply:
Aaaaah, good catch!

Reply:
Oh, I'm sorry, that's not what's happening here. The CLI arg is an env for the upgrade job. The env is passed down from the kubectl-mayastor plugin to the upgrade-job.

Reply:
Oh, I see!
```suggestion
                                value: env::var("HELM_DRIVER").ok(),
```